bazel-mode.el

```diff
@@ -0,0 +1,240 @@
+(defun sort-strings-longest-first (l)
```

Comment:
Please run `M-x checkdoc` on this file and fix the stylistic issues it finds. Please also make sure the file byte-compiles cleanly without warnings.

Reply:
Done.